guides/01_getting-started/03_sharing-your-app.md

```diff
@@ -33,14 +33,19 @@ Share links expire after 72 hours.
 If you'd like to have a permanent link to your Gradio demo on the internet, use Hugging Face Spaces. [Hugging Face Spaces](http://huggingface.co/spaces/) provides the infrastructure to permanently host your machine learning model for free! 
 
 After you have [created a free Hugging Face account](https://huggingface.co/join), you have three methods to deploy your Gradio app to Hugging Face Spaces:
+
 1. From terminal: run `gradio deploy` in your app directory. The CLI will gather some basic metadata and then launch your app. To update your space, you can re-run this command or enable the Github Actions option to automatically update the Spaces on `git push`.
+
 2. From your browser: Drag and drop a folder containing your Gradio model and all related files [here](https://huggingface.co/new-space).
+
 3. Connect Spaces with your Git repository and Spaces will pull the Gradio app from there. See [this guide how to host on Hugging Face Spaces](https://huggingface.co/blog/gradio-spaces) for more information. 
 
 <video autoplay muted loop>
   <source src="/assets/guides/hf_demo.mp4" type="video/mp4" />
 </video>
 
+Note: Some components, like `gr.Image`, will display a "Share" button only on Spaces, so that users can share generated output easily. You can disable this via `gr.Image(shareable=False)`
```

Comment:
```suggestion
Note: Some components, like `gr.Image`, will display a "Share" button only on Spaces, so that users can share the generated output to the Discussions page of the Space easily. You can disable this via `gr.Image(shareable=False)`
```